.gitignore

```diff
@@ -1,3 +1,4 @@
+*.DS_Store
```

Comment:
to keep platform agnostic use a global .gitignore, see https://help.github.com/articles/ignoring-files/#create-a-global-gitignore